lib/index.js

```diff
@@ -7,11 +7,7 @@ const Reporter = require('./reporter');
 const Signal = require('./signal');
 const defaults = {
   server: {
-    url: 'http://localhost:5000',
-    register: {
-      path: 'register',
-      auth: undefined
```

Comment:
How are we planning to do auth? I thought we had talked about sending an object on register?